src/components/TicketStatusFilter.tsx

```diff
@@ -1,27 +1,34 @@
 import { Checkbox, FormControl, InputLabel, MenuItem, Select, SelectChangeEvent, Typography } from '@mui/material';
+import { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useHistory } from 'react-router-dom';
 import { TicketSearchParam } from '../api/searchApi';
 import { TicketStatus, ticketStatusValues } from '../types/publication_types/ticket.types';
 import { dataTestId } from '../utils/dataTestIds';
 
-export const TicketStatusFilter = () => {
+interface TicketStatusFilterProps {
+  isOnTasksPage: boolean;
```

Comment:
like greit å bare sende inn default valgte statuser fra parent her?
```suggestion
  defaultStatusFilter: TicketStatus[];
```

Reply:
Lurt!